src/services/kyc/kycValidatorRewardService.ts

```
@@ -0,0 +1,150 @@
+  onChainTotalMinted: string;
+  rewardAmount: string;
+  maxAllowed: string;
+  reason?: string;
+}
+
+async function getOnChainMintedTotal(): Promise<string> {
+  try {
+    const feeRate = await acbuMintingService.getFeeRate();
+    return feeRate.toString();
+  } catch {
+    logger.warn("Could not read on-chain mint data, falling back to DB aggregate");
```

ESLint

[error] Replace `"Could·not·read·on-chain·mint·data,·falling·back·to·DB·aggregate"` with `⏎······"Could·not·read·on-chain·mint·data,·falling·back·to·DB·aggregate",⏎····`

(prettier/prettier)